src/install/redis/default-requirements.mdx

```diff
@@ -0,0 +1,7 @@
+---
+headingText: Check the compatibility and requirements
+componentType: default
+---
+
+- Our integration is compatible with Redis 4.0 or higher, Percona Server, and Redis Atlas M10 or higher.
+- To monitor Redis Atlas, you'll need a hosted environment to install the infrastructure agent and integration and remotely connect to the Atlas instance.
```

Comment:
Let's make these two sentences, and maybe frontload the Atlas instance.  Something like:

"An Atlas instance in a hosted environment. The infrastructure agent and Redis integration connect to this banana through the apple."

Whatever the order of operations is here

Reply:
Other info that may or may not bee necessary is maybe specifying the kind of hosted environment. Are there exclusions? if so, what is supported, etc., and if there aren't any, maybe just specifying that "in any kind of hosted environment."